hypha/apply/funds/templates/funds/application_base_landing.html

```diff
@@ -15,11 +15,19 @@ <h3>{% blocktrans %}Thank you for your submission to the {{ ORG_LONG_NAME }}.{%
     	{% else %}
         	<p>{% trans "An e-mail with more information has been sent to the address you entered." %}</p>
         {% endif %}
-        <p>{% blocktrans with email=ORG_EMAIL|urlize %}If you do not receive an e-mail within 15 minutes please check your
-spam folder and contact {{ email }} for further assistance.{% endblocktrans %}</p>
+        <p>{% blocktrans with email=ORG_EMAIL|urlize %}If you do not receive an e-mail within 15 minutes please check your spam folder and contact {{ email }} for further assistance.{% endblocktrans %}</p>
+
         {% with email_context=page.specific %}<p>{{ email_context.confirmation_text_extra|urlize }}</p>{% endwith %}
 
         {% block extra_text %}{{ settings.funds.ApplicationSettings.extra_text_round|richtext }}{% endblock %}
+
+		<p>
+		{% if request.user.is_authenticated %}
+			<a href="{% url 'dashboard:dashboard' %}"><button class="button button--primary">{% blocktrans %}Return to my dashboard.{% endblocktrans %}</button></a>
```

Comment:
Looks good. Can we just use 'Go to' instead of 'Return' because return is a kind of reference to the previous location?